homeassistant/components/dyson/sensor.py

```diff
@@ -6,11 +6,11 @@
 """
 import logging
 
+from homeassistant.components.dyson import DYSON_DEVICES, DOMAIN
```

Comment:
All imports from integration component package should be relative. Revert to import as below on line 12 and update that line instead.